tools/release.rb

```diff
@@ -77,7 +77,7 @@ def build_gem
   def build_local
     prepare
 
-    Rake::Task['osc:build'].invoke
+    Rake::Task['osc:build'].invoke("--clean")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.